src/components/apartments/ApartmentsPage.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { getApartments } from '../redux/apartments/apartmentSlice';
```

Comment:
Please change the path to '../../redux/apartments/apartmentSlice' due to error:
Module not found: Error: Can't resolve '../redux/apartments/apartmentSlice' in 'C:\Code\_Projects\apartment_reservation\apartment-reservation-front-end\src\components\apartments'